templates/product-analytics/moose/app/aggregations/dropoff_pages.ts

```diff
@@ -0,0 +1,11 @@
+export default {
+  select: `
+SELECT 
+count() as bounce_count,
```

Comment:
Is that going to work in an aggregation? since this is the Clickhouse engine for aggs